runtime/config.ts

```diff
@@ -19,6 +19,14 @@ export class SkyuxPactConfig {
   public pactProxyServer?: string;
 }
 
+export interface SkyuxConfigUnitTestSettings {
+  browserSet?: 'speedy' | 'quirky' | 'paranoid';
```

Comment:
I like "speedy" and "quirky" :)

Instead of "paranoid", what do you think about "comprehensive" or "exhaustive", to put a more positive spin on it? (If Paul has already approved "paranoid", disregard.)